bindings/wasm/examples/src/merkle_key.js

```diff
@@ -90,9 +90,15 @@ async function merkleKey(clientConfig) {
     console.log(`Identity Update: ${clientConfig.explorer.messageUrl(nextReceipt.messageId)}`);
 
     // Check the verifiable credential is revoked
-    const newResult = await client.checkCredential(signedVc.toString(), VerifierOptions.default());
-    console.log(`VC verification result (false = revoked): ${newResult.verified}`);
-    if (newResult.verified) throw new Error("VC not revoked");
+    let vc_revoked = false; 
+    try {
+        const newResult = await client.checkCredential(signedVc.toString(), VerifierOptions.default());
+    } catch (exception)  {
+        console.log(`VC validation. The credential was revoked. Error message: ${exception.message}`);
+        vc_revoked = true;
```

Comment:
Wouldn't it be better to throw the error here, without using the bool?

Reply:
Here we want the error to occur and I think throwing it will break the CI?